java/runtime/src/main/java/io/ray/runtime/task/ArgumentsBuilder.java

```diff
@@ -49,7 +48,7 @@ public static List<FunctionArg> wrap(Object[] args, Language language) {
             throw new IllegalArgumentException(
                 String.format(
                     "Can't transfer %s data to %s",
-                    Arrays.toString(value.metadata), language.getValueDescriptor().getName()));
+                    new String(value.metadata), language.getValueDescriptor().getName()));
```

Comment:
I don't think we can just pack it in string type as we use raw bytes instead of string in Python: https://github.com/ray-project/ray/blob/master/python/ray/_private/ray_constants.py#L299-L303

One possible solution is that we align these stuff in both Java and Python, for example,
if (meta == b'PYTHON') {
    new String("PYTHON");
} ...